onnxconverter_common/float16.py

```diff
@@ -332,3 +338,57 @@ def convert_float_to_float16_model_path(model_path, min_positive_val=1e-7, max_f
     if not disable_shape_infer:
         model = onnx.load(model_path)
     return convert_float_to_float16(model, min_positive_val, max_finite_val, keep_io_types, disable_shape_infer)
+
+
+def sort_graph_node(graph_proto):
+    # find the "first" node in Nodes that its input is not any node's output
+    def find_first_node(output2node_dict):
+        for node in org_nodes:
+            # if node["name"] not in list_node:
```

Comment:
Is this a comment or debug code? If debug code, could we remove them from the final checkin?